jobs/delorean/2.0/ews/Jenkinsfile

```diff
@@ -0,0 +1,35 @@
+
+pipeline {
+    agent none
+
+    parameters {
+        // Example message:
+        // https://datagrepper.engineering.redhat.com/id?id=ID:messaging-devops-broker01.web.prod.ext.phx2.redhat.com-43656-1547130290231-6:586552:0:0:1&is_raw=true&size=extra-large
```

Comment:
I think it would be better to change this comment to be one of the messages that you actually care about in this job. It's useful for future reference. Here's the most recent one:

https://datagrepper.engineering.redhat.com/id?id=ID:messaging-devops-broker02.web.prod.ext.phx2.redhat.com-44775-1580379079663-8:893046:0:0:1&is_raw=true&size=extra-large